packages/babel-preset-expo/src/web-preset.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright © 2024 650 Industries.
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * A fork of `@react-native/babel-preset` but with everything unrelated to web/ssr removed.
+ * https://github.com/facebook/react-native/blob/2af1da42ff517232f1309efed7565fe9ddbbac77/packages/react-native-babel-preset/src/configs/main.js#L1
+ */
+
+// use `this.foo = bar` instead of `this.defineProperty('foo', ...)`
+const loose = true;
+
+const defaultPlugins = [
+  [require('@babel/plugin-syntax-flow')],
+  [require('babel-plugin-transform-flow-enums')],
+  [require('@babel/plugin-proposal-class-properties'), { loose }],
```

Comment:
```suggestion
  [require('@babel/plugin-transform-class-properties'), { loose }],
```

However, is this needed? There are other removed transforms that imply supporting iOS 14 and up as the cutoff point, and class instance fields were added in iOS 14.5. Being clear on what the cutoff is will be important for anyone using this transform.

Reply:
iOS 13.4 is the cutoff on native. We don't really have definitive rules for the web preset yet.

Reply:
OK - may need the syntax transform for logical assignment operators then